test/driver.js

```diff
@@ -66,6 +66,51 @@ function writeSVG(svgElement, ctx, resolve, reject) {
   };
 }
 
+function inlineImages(images) {
+  const imagePromises = [];
+  for (let i = 0, ii = images.length; i < ii; i++) {
+    const imagePromise = new Promise(function (resolve, reject) {
```

Comment:
Nit: 
```suggestion
    imagePromises.push(new Promise(function (resolve, reject) {
      ...
    }));
```
so you don't need the intermediate `imagePromise` variable.